cmd/soroban-rpc/internal/methods/simulate_transaction.go

```diff
@@ -98,13 +98,14 @@ type SimulateTransactionCost struct {
 }
 
 type InvokeHostFunctionResult struct {
-	XDR string `json:"xdr"`
+	XDR       string   `json:"xdr"`
```

Comment:
Could we rename this field to something a bit more descriptive ?
i.e. "output", "returnValue", "outcome"